smart_card_connector_app/src/chrome-api-provider.js

```diff
@@ -408,12 +408,21 @@ class ChromeEventListener extends goog.Disposable {
               `${nameForLogs}(${argsDebug}): [${resultDebug}]`);
       reportResultFunction(requestId, ...result);
     };
-    this.chromeEvent_.addListener(this.callback_);
+    try {
+      this.chromeEvent_.addListener(this.callback_);
+    } catch (error) {
+      this.callback_ = null;
+      goog.log.warning(
+          logger, `Failed to subscribe to the event ${nameForLogs}: ${error}`);
+    }
   }
```

Comment:
nit: The function is becoming lengthy and hard to read. Consider extracting the lambda (the one that's assigned to `this.callback_` above) to a helper function.

Reply:
done